src/Stratis.Bitcoin.Features.BlockStore/Models/Block.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using NBitcoin;
+using NBitcoin.RPC;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Models
+{
+    public class Block
+    {
+        public string Hash { get; set; }
+        public int Size { get; set; }
+        public int Version { get; set; }
+        public string Bits { get; set; }
+        public DateTimeOffset Time { get; set; }
+        public string[] Transactions { get; set; }
+        public double Difficulty { get; set; }
+        public string MerkleRoot { get; set; }
+        public string PreviousBlockHash { get; set; }
+        public uint Nonce { get; set; }
+
+        public static implicit operator Block(NBitcoin.Block block)
```

Comment:
This operator and the extension method below can be replaced by a ctor taking in an NBitcoin.Block.